showpass/Showpass-E2E-tests
# TA-36-Verify 'My Showpass' page overall appearance

**Description:**
This spec will verifying the following elements on the ‘My Showpass’ page:
- Header ‘My Showpass’
- Subheader ‘Universal QR Code’
- Text under the subheader saying “Use this QR code to scan everything attached to your account!"
- Universal QR Code presence

**Jira ticket:**
https://showpass.atlassian.net/browse/TA-36

**How to test:**
Run the following spec and ensure the corresponding test passes
cypress/e2e/user-settings/my-showpass.cy.js

**Test execution results:**

<img width="825" alt="TA-36-all-tests-passed" src="https://github.com/showpass/Showpass-E2E-tests/assets/138615449/844ccba0-8cc1-4e4b-b173-b71d0a4d6047">
<img width="1721" alt="TA-36-passed" src="https://github.com/showpass/Showpass-E2E-tests/assets/138615449/b35627af-375f-4771-821c-0ec3cf6c7a1e">
